ec-europa/europa-component-library
# feat(links): add MORE> link, refactor template, add documentation - INNO-707

# PR description

Please drop a few lines about the PR: what it does, how to test it, etc.

# QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

- [ ] My PR has the right label(s)
